jujugui/static/gui/src/app/components/canvas-info/canvas-info.js

```diff
@@ -0,0 +1,210 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+/**
+  Displays information for usage and quickstart.
+  Shown when a charm or bundle is added to the canvas, and on deploy.
+*/
+class CanvasInfo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.templateTags = {
+      details_link: {
```

Comment:
Because I'm mapping what's in the markdown file, and that's underscored for readability to userz.